kroxylicious-kms-test-support/src/main/java/io/kroxylicious/kms/service/TestKmsFacadeInvocationContextProvider.java

```diff
@@ -30,6 +32,9 @@ public class TestKmsFacadeInvocationContextProvider implements TestTemplateInvoc
     private static final ExtensionContext.Namespace STORE_NAMESPACE = ExtensionContext.Namespace.create("TEST_KMS");
     private static final String FACADE_FACTORIES = "KMS_FACADE_FACTORIES";
 
+    private static final Pattern FACADE_CLASS_NAME_FILTER = Optional.ofNullable(System.getenv("KMS_FACADE_CLASS_NAME_FILTER")).map(Pattern::compile)
```

Comment:
Is this to enable you to run the tests for just the one implementation you are interested in? Maybe worth adding a note to the class javadoc of the relevant integration tests.